### ISSUE TITLE
DropdownList - keyboard / screen-reader accessibility

### ISSUE THREAD

**Issue by cosminnicula** [jquense/react-widgets#101]
The official specs are listed here http://www.w3.org/WAI/PF/aria-practices/#combobox

- support page up / page down for navigating between pages
- the rw-input is missing an aria-readonly attribute

**Comment by jquense** [comment]
thanks! this is much easier to track

I will definitely have the second bullet done soon, but I do think (for now) I am going to pass on page up/page down, even the reference implementations on the ARIA site don't even implement this (even tho they say it). Hopefully though in the future it will be something I can implement.
